docs/examples/fork-process-model/flask-gunicorn/gunicorn.conf.py

```diff
@@ -50,3 +67,13 @@ def post_fork(server, worker):
         OTLPSpanExporter(endpoint="http://localhost:4317", insecure=True)
     )
     trace.get_tracer_provider().add_span_processor(span_processor)
+
+    reader = PeriodicExportingMetricReader(
+        OTLPMetricExporter(endpoint="http://localhost:4317", insecure=True)
```

Comment:
It's inferred from the scheme already.

```suggestion
        OTLPMetricExporter(endpoint="http://localhost:4317")
```